src/Core/Api/Services/RegisterPurchaseOrderService.php

```diff
@@ -26,20 +28,21 @@ class RegisterPurchaseOrderService extends APIService implements RegisterPurchas
    * @return ResponseDTO
    * @throws \Exception
    */
-  public function register(RequestDTO $requestDTO): ResponseDTO {
+  public function register(RequestDTO $requestDTO): ResponseDTO
+  {
 
     $query = 'mutation purchaseOrders { '
```

Comment:
At some point should we be using a php library for creating graphql queries or are there no good libraries?

Reply:
I just took a quick look and I'm not sure how much value such libraries provide. I understand why you're asking, as the "just keep concatenating strings" thing is less-than-graceful, but for the time being I've been treating it as one of the "if it's not broken, don't fix it" situations.

Perhaps @dFed17 has some insight.

Reply:
Yeah, I definitely wouldn't recommend changing anything just curious if in the future it was worth using to simply and possible get some additional validation that a library could provide.